docs/maui/views/DrawingView.md

```diff
@@ -0,0 +1,133 @@
+    }),
+    LineColor = Colors.Red,
+    LineWidth = 5,
+    BackgroundColor = Colors.Red
+};
+drawingView.DrawingLineCompleted += async (s, e) =>
+{
+    var stream = await e.LastDrawingLine.GetImageStream(gestureImage.Width, gestureImage.Height, Colors.Gray);
+    gestureImage.Source = ImageSource.FromStream(() => stream);
+};
+
+// get stream from lines collection
+var lines = new List<IDrawingLine>();
+var stream1 = await DrawingView.GetImageStream(
+                lines,
+                new Size(gestureImage.Width, gestureImage.Height),
+                Colors.Black);
+
+// get steam from the current DrawingView
+var stream2 = await drawingView.GetImageStream(gestureImage.Width, gestureImage.Height);
+```
+
+## Properties
+
+|Property  |Type  |Description  |
+|---------|---------|---------|
+| Lines | `ObservableCollection<IDrawingLine>` | Collection of `IDrawingLine` that are currently on the `DrawingView` |
+| IsMultiLineModeEnabled | `bool` | Toggles multi-line mode. When true, multiple lines can be drawn on the `DrawingView` while the tap/click is released in-between lines. Note: when `ClearOnFinish` is also enabled, the lines are cleared after the tap/click is released. Additionally, `DrawingLineCompletedCommand` will be fired after each line that is drawn. |
+| ShouldClearOnFinish | `bool` | Indicates whether the `DrawingView` is cleared after releasing the tap/click and a line is drawn. Note: when `IsMultiLineModeEnabled` is also enabled, this might cause unexpected behavior. |
+| DrawingLineCompletedCommand | `ICommand` | This command is invoked whenever the drawing of a line on the `DrawingView` has completed. Note that this is fired after the tap or click is lifted. When `MultiLineMode` is enabled this command is fired multiple times. |
+| DrawingLineCompleted | `EventHandler<DrawingLineCompletedEventArgs>` | `DrawingView` event occurs when drawing line completed. |
+| LineColor | `Color` | The color that is used by default to draw a line on the `DrawingView`. |
+| LineWidth | `float` | The width that is used by default to draw a line on the `DrawingView`. |
+
+### DrawingLine
+
+The `DrawingLine` contains the list of points and allows configuring each line style individually.
+
+#### Properties
+
+|Property  |Type  |Description  |Default value
+|---------|---------|---------|---------|
+| LineColor | `Color` | The color that is used to draw the line on the `DrawingView`. | `Colors.Black` |
+| LineWidth | `float` | The width that is used to draw the line on the `DrawingView`. | `5` |
+| Points | `ObservableCollection<PointF>` | The collection of `PointF` that makes the line. | `new()` |
+| Granularity | `int` | The granularity of this line. Min value is 5. The higher the value, the smoother the line, the slower the program. | `5` |
+| ShouldSmoothPathWhenDrawn | `bool` | Enables or disables if this line is smoothed (anti-aliased) when drawn. | `false` |
+
+#### Custom IDrawingLine
```

Comment:
This could serve as another example in my above suggestion and we show a basic example of a custom implementation